graphql/graphql_jit/server.js

```diff
@@ -0,0 +1,90 @@
+const express = require('express');
+const { buildSchema, parse, execute } = require('graphql');
+const { compileQuery } = require('graphql-jit');
+const bodyParser = require('body-parser');
+
+const schema = buildSchema(`
+  type Post {
+    userId: Int
+    id: Int
+    title: String
+    body: String
+    user: User
+  }
+
+  type User {
+    id: Int
+    name: String
+    username: String
+    email: String
+    phone: String
+    website: String
+  }
+
+  type Query {
+    greet: String
+    posts: [Post]
+    users: [User]
+    post(id: Int!): Post
+    user(id: Int!): User
+  }
+`);
+
+const fetchData = async (url) => {
+  const fetch = (await import('node-fetch')).default;
```

Comment:
We should create a client once and setup its proxy settings to point to nginx.

Reply:
Updated.